synapse/lib/rstorm.py

```diff
@@ -596,6 +601,63 @@ async def _handleStormVcrCallback(self, text):
             raise s_exc.NoSuchCtor(mesg=f'Failed to get callback "{text}"', ctor=text)
         self.context['storm-vcr-callback'] = cb
 
+    async def _handleShell(self, text):
+        '''
+        Execute shell with the supplied arguments.
+        '''
+        parser = argparse.ArgumentParser(add_help=False)
+        parser.add_argument('--include-stderr', action='store_true', help='Include stderr in output.')
+        parser.add_argument('--hide-query', action='store_true', help='Do not include the command in the output.')
+        parser.add_argument('--fail-ok', action='store_true', help='Non-zero return values are non-fatal.')
```

Comment:
Since this is used basically everywhere, it seems like it should be the default? Or shouldn't be used everywhere?

Reply:
It's only used everywhere in this case because the `-h` option to ArgumentParser prints the usage statement and then does a sys.exit(1). I think we generally don't want to allow failures for shell commands. I can bring it up at scrum if you want.